charts/alfresco-activiti/values.yaml

```diff
@@ -61,12 +61,16 @@ persistence:
     mountPath: "/usr/local/data"
     subPath: "alfresco-process-services/process-data"
 readinessProbe: &apsProbe
-  path: /activiti-app/app/rest/locale
+  # -- Make sure to set `readinessProbe.path` to `/activiti-app/app/rest/locale` if you're using a version of APS prior to 24.3.0
+  path: /activiti-app/actuator/health/readiness
   initialDelaySeconds: 25
   periodSeconds: 10
   failureThreshold: 5
   timeoutSeconds: 5
-livenessProbe: *apsProbe
+livenessProbe:
+  <<: *apsProbe
+  # -- Make sure to set `readinessProbe.path` to `/activiti-app/app/rest/locale` if you're using a version of APS prior to 24.3.0
+  path: /activiti-app/actuator/health/liveness
```

Comment:
Shouldn't we have some different initialDelayseconds etc. now when the endpoints are different

Reply:
I see no good settings I could change here. even the initial delay is not worth changing given the app runs in tomcat and tomcat would only treat requests after full startup of the app.